client/app/app.js

```diff
@@ -297,132 +295,24 @@ angular.module("managerApp", [
-                                return val.key === localStorage["univers-selected-language"];
-                            });
-                        }
-                        return selectedLang ? selectedLang.name : $translate.instant("common_menu_language");
-                    })(),
-                    subLinks: (function () {
-                        var subLinksLang = [];
-                        angular.forEach(LANGUAGES.available, function (lang) {
-                            subLinksLang.push({
-                                label: lang.name,
-                                click: function () {
-                                    localStorage["univers-selected-language"] = lang.key;
-                                    window.location.reload();
-                                },
-                                lang: _.chain(lang.key).words().head().value()
-                            });
-                        });
-                        return subLinksLang;
-                    })()
-                }, {
-                    label: user.firstname + " " + user.name + " (" + user.nichandle + ")",
-                    subLinks: [
-                        {
-                            label: $translate.instant("common_menu_account"),
-                            url: REDIRECT_URLS.userInfos
-                        }, {
-                            label: $translate.instant("common_menu_billing"),
-                            url: REDIRECT_URLS.billing
-                        }, {
-                            label: $translate.instant("common_menu_renew"),
-                            url: REDIRECT_URLS.services
-                        }, {
-                            label: $translate.instant("common_menu_orders_all"),
-                            url: REDIRECT_URLS.orders
-                        }, {
-                            label: $translate.instant("common_menu_orders"),
-                            url: "#/orders"
-                        }, {
-                            label: $translate.instant("global_logout"),
-                            click: function () {
-                                ssoAuthentication.logout();
-                            }
-                        }
-                    ]
-                }
-            ]);
-
-            managerNavbar.setCurrentLink({
-                universe: "telecom",
-                label: $translate.instant("common_menu_telecom"),
-                url: MANAGER_URLS.telecom
-            });
-        });
+        // Get structure of the navbar
+        $translate.refresh().then(() =>
+            ManagerNavbarService.getNavbar().then((navbar) => {
+                $rootScope.navbar = navbar;
+            })
+        );
+
+        // Scroll to anchor id
+        $rootScope.scrollTo = (id) => {
+            // Set focus to target
+            $document.find(`#${id}`)[0].focus();
```

Comment:
is `$document` nessary here ? Can't we use directly `$` of jQuery ?

Reply:
Not a fan of using jQuery with AngularJS. 
But for this case, I could go with a simple `$document[0].getElementById(id).focus()`